src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/SblInstanceEvent.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Models
+{
+    /// <summary>
+    /// Model for the instance event for interaction with the SBL solution
+    /// </summary>
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class SblInstanceEvent
```

Comment:
were we going to simplify this class? i see sonarCloud is complaining about duplication. My suggestion would be to keep the following fields: instanceId, createdDateTime, userId and endUserSystemId for now.